src/shared/components/form/formSelect/formSelect.js

```diff
@@ -2,6 +2,18 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 export default class FormSelect extends Component {
+  onChange(e) {
+    // make sure we have a valid value before propogating the on change call
+    let isValid = true;
+    if (this.props.validationFunc) {
+      isValid = this.props.validationFunc(e);
+    }
+
+    if (isValid && this.props.onChange) {
```

Comment:
Same question here. Checking `isValid` makes perfect sense. Did we have a scenario where `onChange` was undefined?

Reply:
Same as previous answer. In some cases we might choose not to implement an `onChange` function.